config/auth.php

```diff
@@ -110,4 +115,12 @@
 
     'password_timeout' => 10800,
 
+
+    'login_parameters' => [
+        'locale' => 'nl',
+        'referred' => 'publiq-platform',
+        'prompt' => 'login',
+        'skip_verify_legacy' => 'true',
+        'product_display_name' => 'publiq platform',
+    ],
```

Comment:
Is there a way to store these in a config file that will be stored inside the `cultuurnet/appconfig` repo instead? Since that is the easiest place for non-developers to configure them

Reply:
When storing an object it is indeed required to have multiple env variables to match all the properties. This was done here: https://github.com/cultuurnet/publiq-platform/blob/246556222043248ac1301381048f1cde7a28c29e/config/insightly.php

That makes it possible for non-developers to configure the object inside the env file. But that is not a solution for adding and removing login parameters.

The config file is pure PHP so we can always create a helper method to parse other files and take the values from other files than just `.env` files.

Reply:
Yep we can add another PHP file for the login parameters and include that. Just a bit sad how we then end up with a PHP (app)config file + an ENV (app)config file, instead of just one PHP (app)config file or one ENV (app)config file. It's like a weird compromise to use an ENV file but also use a PHP file for the things that the ENV file cannot solve. 😅 But it's a nit and fine for me.

Reply:
In theory, we could work with config files only.

The only requirement for the `.env` file is the MySQL database values used by Docker to create a container with a pre-created database. We could workaround this by creating a custom image with predefined values.

For now, I'm gonna keep it as in the scope of this ticket, but I created https://jira.uitdatabank.be/browse/PPF-70 to do some extra investigation

Reply:
Aha okay now I see why we'd need an `.env` file. In that case let's keep using that and just try to find a way to make the list of login parameters (both the keys & values) configurable by non-devs in the appconfig. For example with another separate PHP config file indeed.